queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM public.animals WHERE name LIKE '%mon';
+
+SELECT name FROM public.animals WHERE date_of_birth BETWEEN '01-01-2016' AND '12-31-2019';
+
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
```

Comment:
**Changes Required**
- [ ] Currently the query is invalid with TRUE. It should be `lowercase and in single or double qoutes`. Let us try to fix this bug.

Reply:
![Captura de pantalla 2021-09-21 a la(s) 7 23 30 p  m](https://user-images.githubusercontent.com/83717285/134264665-23e48a2a-9c46-4042-a7ea-47ef78a7cfd1.png)

Hi @oxenprogrammer, 

thank you for your suggestions, I used `true` without quotes to avoid this error, I made the rest of the changes you suggested.